science/molmol/Portfile

```diff
@@ -50,6 +52,9 @@ post-patch {
     if {${os.major} == 12} {
       reinplace  "s|SYSLIB = |SYSLIB = -isysroot `xcode-select --print-path`/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.7.sdk |g" ${worksrcpath}/makedef
     }
+    if {${os.major} >= 20} {
+      reinplace  "s|/System/Library/Frameworks/OpenGL.framework/Libraries/libGL.dylib ||g" ${worksrcpath}/makedef
```

Comment:
Not sure completely removing this would be correct as while OpenGL had been deprecated  for years now even the latest SDKs still provide OpenGL.framework with working tbd stubs.